src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/CounterGroup.cs

```diff
@@ -0,0 +1,247 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file got separated out from EventCounter.cs. I made a comment on the only place that it changed.